openreview/matching.py

```diff
@@ -8,16 +8,24 @@ def get_conflicts(author_profiles, user_profile):
     authors_relation_conflicts = set()
     conflicts = set()
 
-    for p in author_profiles.values():
-        author_domain_conflicts, author_relation_conflicts = get_profile_conflicts(p)
+    author_emails = author_profiles.keys()
+    user_emails = user_profile.content['emails']
 
-        authors_domain_conflicts.update(author_domain_conflicts)
-        authors_relation_conflicts.update(author_relation_conflicts)
+    for author_email, profile in author_profiles.iteritems():
+        if profile:
```

Comment:
this is the first main change. Before, a dict of profiles keyed by emails was being passed in. Now, it can handle a dict keyed by emails where some of the values are profiles, and some are None (if, for example, the author doesn't have a profile).